Makefile

```diff
@@ -16,7 +16,7 @@ build:
 	go install -ldflags "-X main.version=$(VERSION)" ./...
 
 build-windows:
-	go build -o telegraf.exe -ldflags \
+	GOOS=windows GOARCH=386 go build -o telegraf.exe -ldflags \
```

Comment:
I derived the value `386` from `/scripts/build.py`.

On Wed, Aug 10, 2016, 2:58 PM Cameron Sparr <notifications@github.com>
wrote:

> In Makefile
> <https://github.com/influxdata/telegraf/pull/1621#discussion_r74339009>:
>
> > @@ -16,7 +16,7 @@ build:
> >  	go install -ldflags "-X main.version=$(VERSION)" ./...
> >
> >  build-windows:
> > -	go build -o telegraf.exe -ldflags \
> > +	GOOS=windows GOARCH=386 go build -o telegraf.exe -ldflags \
>
> why 386? are windows servers still 32-bit?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/influxdata/telegraf/pull/1621/files/3115d751b00c6bdabc3795f5f71c9dac918093c3#r74339009>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABGK3xw1rzcOgpCDOic7QjOHhikFvE_tks5qekmMgaJpZM4Jhf2Q>
> .
>